### PR TITLE
Update package.json to fix Regular Expression Denial of Service in debug

### PR DESCRIPTION
**Pull request by martinez-hugo**

### CODE REVIEW

**Comment by cclauss** [comment]
Please insert the line `- npm run update-crosswalk` after line 26 of the file `appveyor.yml` so we can see if your tests pass.

**Comment by cclauss** [comment]
Please rebase or close this pull request.

File: test/proxy-bcrypt.test.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const { createUnzip } = require('zlib');
 const os = require('os');
 
 const tar = require('tar-fs');
-const Agent = require('https-proxy-agent');
+const { HttpsProxyAgent } = require('https-proxy-agent');
```
Comment (benmccann) [inline test/proxy-bcrypt.test.js:9]: if this is being updated, wouldn't you also have to update `lib/install.js`?

**Review by benmccann — CHANGES_REQUESTED** [review]

**Comment by benmccann** [comment]
closing in favor of https://github.com/mapbox/node-pre-gyp/pull/747

**Pull request closed without merging**